src/fswAlgorithms/imageProcessing/centerOfBrightness/_UnitTest/test_centerOfBrightness.py

```diff
@@ -0,0 +1,186 @@
+ # OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+
+import pytest
+import os, inspect, glob
+import numpy as np
+
+filename = inspect.getframeinfo(inspect.currentframe()).filename
+path = os.path.dirname(os.path.abspath(filename))
+bskName = 'Basilisk'
+splitPath = path.split(bskName)
+
+# Import all of the modules that we are going to be called in this simulation
+importErr = False
+reasonErr = ""
+try:
+    from PIL import Image, ImageDraw
+except ImportError:
+    importErr = True
+    reasonErr = "python Pillow package not installed---can't test CenterOfBrightness module"
+
+# Import all of the modules that we are going to be called in this simulation
+from Basilisk.utilities import SimulationBaseClass, unitTestSupport
+from Basilisk.utilities import macros
+from Basilisk.architecture import messaging
+
+try:
+    from Basilisk.fswAlgorithms import centerOfBrightness
+except ImportError:
+    importErr = True
+    reasonErr = "Center Of Brightness not built---check OpenCV option"
+
+# Uncomment this line is this test is to be skipped in the global unit test run, adjust message as needed.
+# @pytest.mark.skipif(conditionstring)
+# Uncomment this line if this test has an expected failure, adjust message as needed.
+# @pytest.mark.xfail(conditionstring)
+# Provide a unique test method name, starting with 'test_'.
+
+@pytest.mark.skipif(importErr, reason= reasonErr)
+@pytest.mark.parametrize("image, blur,  saveTest, valid, saveImage", [
+                    ("full_circle.png",   1, False, 1,  False)
+                    ,("test_circle.png",  1, False, 1,  False)
+                    ,("half_half.png",    1, True , 1,  False)
+])
+
+# update "module" in this function name to reflect the module name
+def test_module(show_plots, image, blur, saveTest, valid, saveImage):
+    """
+    Unit test for Center of Brightness algorithm. The unit test specifically runs on 2 images:
+
+        1. A crescent Mars: This image only contains a slim Mars crescent
+        2. Moons: This image contains several Moon crescents
+
+    This modules compares directly to the expected circles from the images.
+    """
+    # each test method requires a single assert method to be called
+    [testResults, testMessage] = cobTest(show_plots, image, blur, saveTest, valid, saveImage)
+    assert testResults < 1, testMessage
+
+
+def cobTest(show_plots, image, blur, saveTest, valid, saveImage):
+
+    # Truth values from python
+    imagePath = path + '/' + image
+    input_image = Image.open(imagePath)
+    input_image.load()
+    #################################################
+
+    testFailCount = 0                       # zero unit test result counter
+    testMessages = []                       # create empty array to store test log messages
+    unitTaskName = "unitTask"               # arbitrary name (don't change)
+    unitProcessName = "TestProcess"         # arbitrary name (don't change)
+
+    # Create a sim module as an empty container
```

Comment:
:large_orange_diamond: My preference is to remove all of these type of comments from the test. They clutter the view and add nothing to clarity. I know these are from copy and paste

Reply:
Done